pls/rootchain_manager.go

```diff
@@ -676,7 +676,7 @@ func (rcm *RootChainManager) handleBlockFinalzied(ev *rootchain.RootChainBlockFi
 		Context: context.Background(),
 	}
 
-	w, err := rcm.accountManager.Find(rcm.config.Operator)
+	w, err := rcm.accountManager.Find(rcm.config.Challenger)
```

Comment:
What this changed?

Reply:
Now that we use challenger account instead of operator account in case of exitChallenge, we need wallet object related with challenger account, not the operator account. If it is not changed, the following line incurs `err: unknown account`
https://github.com/Onther-Tech/plasma-evm/blob/fix-rcm-test/pls/rootchain_manager.go#L711

Reply:
Then it should also change the below log.
`log.Error("Failed to get operator wallet", "err", err)`